src/components/typography/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import { Slot } from "@radix-ui/react-slot"
+import { cva, type VariantProps } from "class-variance-authority"
+import { cn } from "../../lib/utils"
+
+export const typographyVariants = cva("", {
+  variants: {
+    variant: {
+        h1: "scroll-m-20 text-4xl font-semibold tracking-[-0.04em] lg:text-5xl",
```

Comment:
what's up with tracking-[-0.04em] why do we even care about it ? I also don't see such value in a design guide.